x-pack/test/saved_object_tagging/functional/tests/visualize_integration.ts

```diff
@@ -159,33 +198,43 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await listingTable.waitUntilTableIsLoaded();
 
         await selectFilterTags('my-new-tag');
-        const itemNames = await listingTable.getAllItemsNames();
+        const itemNames = await listingTable.getAllSelectableItemsNames();
         expect(itemNames).to.contain('vis-with-new-tag');
       });
     });
 
-    // FLAKY: https://github.com/elastic/kibana/issues/88639
-    describe.skip('editing', () => {
-      beforeEach(async () => {
+    describe('editing', () => {
+      before(async () => {
         await PageObjects.visualize.gotoVisualizationLandingPage();
-        await listingTable.waitUntilTableIsLoaded();
+        await PageObjects.visualize.deleteAllVisualizations();
+        // create a vis to add a tag to during edit
+        await PageObjects.visualize.navigateToNewVisualization();
+        await createSimpleMarkdownVis({
+          visText: 'Edit me!',
+          visName: 'MarkdownViz',
+        });
       });
 
       it('allows to assign tags to an existing visualization', async () => {
-        await listingTable.clickItemLink('visualize', 'Visualization 1 (tag-1)');
-
+        await PageObjects.visualize.gotoVisualizationLandingPage();
+        await listingTable.waitUntilTableIsLoaded();
+        await listingTable.clickItemLink('visualize', 'MarkdownViz');
         await PageObjects.visualize.ensureSavePanelOpen();
-        await selectSavedObjectTags('tag-2');
-
+        await selectSavedObjectTags(...['myextratag']);
```

Comment:
nit: do we need the spread?
```suggestion
        await selectSavedObjectTags('myextratag');
```